### PR TITLE
docs: add "Modifying the Head Element" example

### LINKED ISSUE #1215: proposal(docs): add an example of how to modify `<head />` element
I.e. using the `<Head />` element.

### PR DESCRIPTION
**Pull request by DanielXing1103**
resolves #1215 by adding example on modifying ``` <Head />``` on index.md

### CODE REVIEW

**Review by iuioiua — CHANGES_REQUESTED** [review]
I like the conciseness of this. Though, I think the title should be "Modifying the `<head>` Element" and the example should have its own `modifying-the-head-element.md` file in the `/docs/examples` folder. We can provide a link to the page on the examples page like https://deno.com/manual@v1.34.1/examples

Another title idea is "Modifying Web Page Metadata via the `<head>` Element".

**Comment by DanielXing1103** [comment]
Thank you very much for the feedback. Just to clarify, is the example in this PR the one mentioned in #1215?

**Comment by iuioiua** [comment]
> Just to clarify, is the example in this PR the one mentioned in #1215?

Yes, it is 👍🏾

File: docs/examples/modifying-the-head-element.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Modifying Web Page Metadata via the ```<Head /> ```Element
```
Comment (iuioiua) [inline docs/examples/modifying-the-head-element.md]:
```suggestion
# Modifying Web Page Metadata via the `<Head />` Element
```
Triple backticks aren't needed for inline code snippets. Can you please use a single backtick where appropriate?

File: docs/examples/modifying-the-head-element.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Modifying the `<head>` Element
+
+We can use the `<Head />` component in `$fresh/runtime.ts` to modify the `<head>`
+element. For example:
+
+```tsx
+import { Head } from "$fresh/runtime.ts";
```
Comment (iuioiua) [inline docs/examples/modifying-the-head-element.md:7]:
```suggestion
// routes/index.tsx
import { Head } from "$fresh/runtime.ts";
```

File: docs/examples/image/Head.png
Comment (iuioiua) [inline docs/examples/image/Head.png:1]: This image is not needed.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Few thoughts:
- The article should have the paragraph text come first, then the snippet last.
- Can we provide a short list of some common uses for modifying the `<head>` element?
- The `<Head />` component only includes a `<title>` element. Can we add a few more common use cases, such as meta tags, script, etc.?

File: docs/examples/modifying-the-head-element.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <img
+          src="/logo.svg"
+          class="w-32 h-32"
+          alt="the fresh logo: a sliced lemon dripping with juice"
+        />
+        <p class="my-6">
+          Welcome to `fresh`. Try updating this message in the
+          ./routes/index.tsx file, and refresh.
+        </p>
+        <Counter start={3} />
```
Comment (lucacasonato) [inline docs/examples/modifying-the-head-element.md]:
```suggestion
        <h1>Hello World</h1>
```

This part of the example is not important to what we are trying to bring across, so we can simplify. This allows users to focus on the important bit (the `<Head>` tag).

File: docs/examples/index.md [hunk 4]
```diff
@@ -7,3 +7,5 @@ In this chapter of the Fresh documentation, you can find examples of features
 that you may like in your Fresh project. If there's a specific example you'd
 like to see here, please open
 [a GitHub discussion](https://github.com/denoland/fresh/discussions/new?category=ideas).
+
+- [Modifying Web Page Metadata via the `<head>` Element](modifying-the-head-element.md)
```
Comment (iuioiua) [inline docs/examples/index.md]:
```suggestion
- [Modifying the `<head>` Element](modifying-the-head-element)
```

File: docs/examples/modifying-the-head-element.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Modifying the `<head>` Element
+
+We can use the `<Head />` component in `$fresh/runtime.ts` to modify the
+`<head>` element. By adding changes to the inner components of `<Head />` we can
+add a new components to the `<head />` element of the display page.
```
Comment (iuioiua) [inline docs/examples/modifying-the-head-element.md]:
```suggestion
add a new components to the `<head>` element of the display page.
```

File: docs/examples/modifying-the-head-element.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Common components for modifying `<head>`
+
+- `<title>` Setting the document title
+- `<meta>` Specifying metadata
+- `<link>` Link external resources
+- `<script>` reference JavaScript code
```
Comment (iuioiua) [inline docs/examples/modifying-the-head-element.md]:
```suggestion
## Common uses for modifying `<head>`

- Setting the document title using `<title>`
- Specifying page metadata using `<meta>`
- Linking to external resources using `<link>`
- Including JavaScript code via `<script>`
```

**Review by iuioiua — CHANGES_REQUESTED** [review]

**Review by lucacasonato — APPROVED** [review]
Great, TY!

**Review by iuioiua — APPROVED** [review]
LGTM! Thanks, @DanielXing1103.

**Comment by DanielXing1103** [comment]
Thank you very much for the help along the way.

**Pull request merged**